Resources/Textures/_CD/Clothing/Shoes/Specific/grayskates.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by brainfood1183 (github), altered by discord: mrgreen06 (358779039810322433)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "icon"
```

Comment:
Make sure your file names are properly renamed as per the meta, so 'Icon' should be 'icon'